### PR TITLE
Don't use already processed propers if we do a forced proper search

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

File: sickbeard/properFinder.py [hunk 1]
```diff
@@ -83,9 +83,9 @@ def run(self, force=False):  # pylint: disable=unused-argument
             run_at = ', next check in approx. {0}'.format(
                 '{0}h, {1}m'.format(hours, minutes) if 0 < hours else '{0}m, {1}s'.format(minutes, seconds))
 
-        # Restore processed propers and add new ones to the end
+        # Restore processed propers and add new ones to the beginning
```
Comment (fernandog) [inline sickbeard/properFinder.py]: @medariox why begining? when we add items to list we add in the end right?

File: sickbeard/properFinder.py [hunk 2]
```diff
@@ -83,9 +83,9 @@ def run(self, force=False):  # pylint: disable=unused-argument
...
         if force:
-            self.processed_propers = list(set(self.processed_propers).difference(set(current_processed_propers))) + current_processed_propers
+            self.processed_propers[:0] = set(self.processed_propers).difference(set(current_processed_propers))
```
Comment (fernandog) [inline sickbeard/properFinder.py]: @medariox what [:0] does?

Comment (medariox) [inline sickbeard/properFinder.py]: Slicing. Here it is explained quite well: http://www.diveintopython.net/native_data_types/lists.html

**Comment by medariox** [comment]
Approved

**Pull request merged**